src/ngx-smart-modal/src/components/ngx-smart-modal.component.ts

```diff
@@ -23,8 +23,9 @@ import { NgxSmartModalConfig } from '../config/ngx-smart-modal.config';
          [ngClass]="{'transparent':!backdrop, 'overlay':true, 'nsm-overlay-open':openedClass}"
          (mousedown)="dismiss($event)" #nsmOverlay>
       <div [style.z-index]="visible ? layerPosition : -1"
-           [ngClass]="['nsm-dialog', customClass, openedClass ? 'nsm-dialog-open': 'nsm-dialog-close']" #nsmDialog>
-        <div class="nsm-content" #nsmContent>
+           [ngClass]="['nsm-dialog', customClass, openedClass ? 'nsm-dialog-open': 'nsm-dialog-close']"
+           [style.position]="draggable?'absolute':'relative'" #nsmDialog>
```

Comment:
Sry for didn't saw it before, add some space before and after `?` and `:` like other lines